### PR TITLE
Document Windows installation

### PR DESCRIPTION
**Pull request by silverbacknet**
The Windows installation is surprisingly simple, at least with VS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +58,18 @@ heif_file = pyheif.read("IMG_7424.HEIC")
...
 ```
 
+### Windows DLL Loading
+
+On Python 3.8+, heif.dll and its dependencies must be copied to the site-packages folder, or explicity located in script:
+
+```python
+import os
+os.add_dll_directory(r'X:\path\to\libheif\build')
+import libheif
```
Comment (homm) [inline README.md:68]: How is this possible to import `libheif` directly? Is it `pyheif`?

Comment (silverbacknet) [reply README.md]: Ooof, yes it should be pyheif; that was a lousy typo I made while genericizing it. Sorry.

**Comment by cdgriffith** [comment]
Tried this and not building for me, but could have missed something obvious. 

```
git clone https://github.com/microsoft/vcpkg
.\vcpkg\bootstrap-vcpkg.bat -disableMetrics
.\vcpkg\vcpkg install libheif
.\vcpkg\vcpkg integrate install

# System restart

set LIB="C:\Users\Chris\projects\vcpkg\packages\libheif_x86-windows\lib"
set INCLUDE="C:\Users\Chris\projects\vcpkg\packages\libheif_x86-windows\include"

pip install pyheif --user
```

Errors with: 

```
Installing collected packages: pyheif
  Running setup.py install for pyheif ... error
  error: subprocess-exited-with-error

  × Running setup.py install for pyheif did not run successfully.
  │ exit code: 1
  ╰─> [9 lines of output]
      running install
      C:\Program Files\Python310\lib\site-packages\setuptools\command\install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
        warnings.warn(
      running build
      running build_py
      running build_ext
      generating cffi module 'build\\temp.win-amd64-3.10\\Release\\_libheif_cffi.c'
      building '_libheif_cffi' extension
      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure
```

I have all the build tools, and just installed fresh x86 and 64 redists to be sure as well. 
build_tools

I also tried without the quotes around the lib / include and then the linker doesn't find what's needed: 
```
    Creating library build\temp.win-amd64-3.10\Release\build\temp.win-amd64-3.10\Release\_libheif_cffi.lib and object build\temp.win-amd64-3.10\Release\build\temp.win-amd64-3.10\Release\_libheif_cffi.exp
    _libheif_cffi.obj : error LNK2001: unresolved external symbol __imp_heif_get_version_number_minor
    _libheif_cffi.obj : error LNK2001: unresolved external symbol __imp_heif_image_handle_release
    ...
    build\lib.win-amd64-3.10\_libheif_cffi.pyd : fatal error LNK1120: 71 unresolved externals
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools\\VC\\Tools\\MSVC\\14.29.30133\\bin\\HostX86\\x64\\link.exe' failed with exit code 1120
```

**Comment by silverbacknet** [comment]
You wiped out your compiler's LIB and INCLUDE by erasing the %LIB%; and %INCLUDE%; in those lines. It should also not be quoted, though I think it might be able to handle that anyway.

**Comment by silverbacknet** [comment]
Sorry, I didn't see edit 2. Hm. Good question. Both x86 and x64 toolchains can cross-compile, but it'd still be necessary to set a specific target.

**Pull request closed without merging**